net/qpid-proton/Portfile

```diff
@@ -19,14 +21,15 @@ homepage            https://qpid.apache.org
 
 PortGroup           github 1.0
 
-github.setup        apache qpid-proton 0.29.0
-revision            1
-
 PortGroup           cmake 1.1
 
-checksums           rmd160  7bcee7f9a1fe7b9aef5bc1e914eed05e72411e26 \
-                    sha256  38d64df3d99020a636780fa8d658b217b94717d4451ae5f65a8def18304c8c7d \
-                    size    1258304
+checksums           rmd160  51a80f8858885f76b506b50ea6a12cfab84dca5e \
+                    sha256  c1bf2618b9e48c3033a3d5713c7fd6018f3b3b15d3bad90bf00d647c8c21b40f \
+                    size    1316310
+
+cmake.build_type    RelWithDebInfo
```

Comment:
@mf2k Is the location and syntax here ok?

Reply:
That should be fine.